spec/accounts_spec.rb

```diff
@@ -54,7 +54,8 @@
             "name" => "ACME Inc"
           }
         ],
-        "has_more" => false
+        "has_more" => false,
+        "url" => "/accounts"
```

Comment:
How about next_url

Reply:
hmm yeah, should have added that as a null in the test